src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesAndValues = sourceString.split(';');
```

Comment:
Instead of using a semicolon to split the string, consider using a regular expression that matches semicolons followed by any amount of whitespace. This would help to avoid the need for trimming each property and value pair later on.